source/builders.txt

```diff
@@ -0,0 +1,117 @@
+.. _kotlin-sync-builders:
+
+===============================
+Implement Builders Code Pattern
+===============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. .. toctree::
+
+..    /fundamentals/builders/aggregates
+..    /fundamentals/builders/filters
+..    /fundamentals/builders/indexes
+..    /fundamentals/builders/projections
+..    /fundamentals/builders/sort
+..    /fundamentals/builders/updates
+
+Overview
+--------
+
+This page describes how to implement the builders pattern in your code and describes
```

Comment:
Nit: This seems a little verbose. In short its about using builders to help build operations in a less verbose way than using Documents.

Reply:
For further clarification, do you think the entire Overview section is too verbose?

Reply:
Will add suggesions